src/bundle-configuration/zookeeper-prod/runtime-config.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+export ZK_DATA_DIR=/var/lib/zookeeper
```

Comment:
@markusjura and @huntc - I think we should follow the default directory provided by Zookeeper for version `3.5.2` which is `/var/lib/zookeeper`.

The getting started docs for Standalone setup has `/var/lib/zookeeper` under `Standalone Operation`:

http://zookeeper.apache.org/doc/r3.5.2-alpha/zookeeperStarted.html

Also for Administrator's guide for setting up clustered environment has `/var/lib/zookeeper` under `Clustered (Multi-Server) Setup`:

http://zookeeper.apache.org/doc/r3.5.2-alpha/zookeeperAdmin.html#var_datadir

I'd suggest we keep the convention of matching what the expected defaults of the bundle.